src/Symfony/Bridge/Doctrine/SchemaListener/RememberMeTokenProviderDoctrineSchemaSubscriber.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\SchemaListener;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\ORM\Tools\Event\GenerateSchemaEventArgs;
+use Doctrine\ORM\Tools\ToolEvents;
+use Symfony\Bridge\Doctrine\Security\RememberMe\DoctrineTokenProvider;
+use Symfony\Component\Security\Http\RememberMe\PersistentRememberMeHandler;
+use Symfony\Component\Security\Http\RememberMe\RememberMeHandlerInterface;
+
+/**
+ * Automatically adds the remember me table needed for the {@see DoctrineTokenProvider}.
+ *
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
+final class RememberMeTokenProviderDoctrineSchemaSubscriber implements EventSubscriber
```

Comment:
> Furthermore, a schema listener is created. Whenever the doctrine token provider is used, 
> make:migration/doctrine:schema:update will automatically create the required table.

I think this is not true quite yet - I don't think this class is registered as a service anywhere - so I don't get any automatic schema updates after enabling Doctrine.

Reply:
Registering it is DoctrineBundle's job, see https://github.com/doctrine/DoctrineBundle/pull/1297

Reply:
doh! Of course! I will bring in that PR to my project to keep testing

Reply:
It's working nicely with that PR :)